packages/explat-client/src/internal/experiment-assignments.ts

```diff
@@ -0,0 +1,10 @@
+import { ExperimentAssignment } from '../types';
+import * as Timing from './timing';
+
+export function isAlive( experimentAssignment: ExperimentAssignment ) {
```

Comment:
I didn't comment this one, it just checks if an ExperimentAssignment is within the TTL.